src/main/perl/VXVM.pm

```diff
@@ -0,0 +1,86 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+# ${build-info}
+
+=pod
+
+=head1 VXVM
+
+This class defines a VXVM volume. It is part of the
+blockdevices framework.
+
+=cut
+
+package NCM::VXVM;
+
+use strict;
+use warnings;
+
+use NCM::Blockdevices;
+use EDG::WP4::CCM::Element qw (unescape);
+
+our @ISA = qw{NCM::Blockdevices};
+
+our $this_app = $main::this_app;
+
+sub _initialize
+{
+    my ($self, $path, $config) = @_;
+
+    my $st = $config->getElement($path)->getTree;
+    $path =~ m!/([^/]+)$!;
```

Comment:
should be 
```perl
if ($path =~ m!/([^/]+)$!) {
    $self->{devname} = unescape($1);
} else {
    $this_app->error("cannot determine devname from $path");
}
```

wherever you copied this from, maybe also clean it up there :smile: